### PR TITLE
chore: FPLA-2360 Remove SDO for SN20C50019

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-2360

### Change description ###
Remove SDO for SN20C50019

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java [hunk 1]
```diff
@@ -34,31 +30,16 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         Map<String, Object> data = caseDetails.getData();
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
-        if ("SA20C50034".equals(caseData.getFamilyManCaseNumber()) && 1600893083735118L == caseDetails.getId()) {
-            data.put("hearingDetails", removeHearingLinkedToCmo(caseData.getHearingDetails(),
-                caseData.getSealedCMOs().get(0).getId()));
+        if ("SN20C50019".equals(caseData.getFamilyManCaseNumber()) && 1603717767912577L == caseDetails.getId()) {
+            data.remove("standardDirectionOrder");
+            data.remove("noticeOfProceedingsBundle");
```
Comment (Jamesnelson117) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java:35]: Hmm, do we definitely want to remove NOP's? As they were created via a separate event prior to creating SDO. I think we are right in removing them but it's worth checking with Gail / James to confirm this is the right thing to do.

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java [hunk 2]
```diff
@@ -34,31 +30,16 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         Map<String, Object> data = caseDetails.getData();
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
-        if ("SA20C50034".equals(caseData.getFamilyManCaseNumber()) && 1600893083735118L == caseDetails.getId()) {
-            data.put("hearingDetails", removeHearingLinkedToCmo(caseData.getHearingDetails(),
-                caseData.getSealedCMOs().get(0).getId()));
+        if ("SN20C50019".equals(caseData.getFamilyManCaseNumber()) && 1603717767912577L == caseDetails.getId()) {
```
Comment (Jamesnelson117) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java:33]: Checking both is probably a bit unnecessary moving forward as we can guarantee that CCD ID is unique to case. But happy if you feel safer with both considering we are modifying prod data.

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java (hunk 1 above)
Comment (aliveni) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java]: updated to remove just the standardDirectionOrder from case data

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java (hunk 2 above)
Comment (aliveni) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java]: Removed condition on FamilyManCaseNumber

**Review by nkoranne — COMMENTED** [review]
LGTM

**Review by nkoranne — APPROVED** [review]

**Pull request merged**